src/react-native/11-navigation/navigation.md

```diff
@@ -14,7 +14,12 @@ In this section, you will:
 
 ## Objective 1: Using React Native bottom tab navigation
 
-TODO
+<div style="display: flex; flex-direction: row; gap: 2rem">
+  <img alt="Screen Shot of the StateList in Navigation" src="../static/img/react-native/11-navigation/NavigationStateList.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
+  <img alt="Screen Shot of the CityList in Navigation" src="../static/img/react-native/11-navigation/NavigationCityList.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
+  <img alt="Screen Shot of the RestaurantList in Navigation" src="../static/img/react-native/11-navigation/NavigationRestaurantList.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
+  <img alt="Screen Shot of the RestaurantDetails in Navigation" src="../static/img/react-native/11-navigation/NavigationRestaurantDetails.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
+</div>
```

Comment:
This goes off to the right of the screen, so the CSS would need to change to keep all of them.

I think only the first one is needed for this first objective, and the others can be used for the later objectives.